Gem/Code/Source/Components/NetworkPlayerMovementComponent.cpp

```diff
@@ -132,16 +170,55 @@ namespace MultiplayerSample
         GetEntity()->GetTransform()->SetLocalRotationQuaternion(newOrientation);
 
         // Update velocity
-        UpdateVelocity(*playerInput);
+        UpdateVelocity(*playerInput, deltaTime);
 
-        GetNetworkCharacterComponentController()->TryMoveWithVelocity(GetVelocity(), deltaTime);
+        // absolute velocity is based on velocity generated by the player and other sources
+        const AZ::Vector3 absoluteVelocity = GetBaseVelocity() + GetLocalVelocity();
+
+        // if we're not moving down on a platform and have a negative velocity we're falling
+        GetNetworkAnimationComponentController()->ModifyActiveAnimStates().SetBit(
+            aznumeric_cast<uint32_t>(CharacterAnimState::Falling), !onGround && absoluteVelocity.GetZ() < 0.f);
+
+        GetNetworkCharacterComponentController()->TryMoveWithVelocity(absoluteVelocity, deltaTime);
+
+        m_wasOnGround = onGround;
```

Comment:
It might be good to make this an Authority to Autonomous predictable network property? The transform might be sufficient on its own however, so I'm not certain if you would ever need the contact state to be corrected such that correcting the transform doesn't provide the proper value.. That would imply some sort of randomness from the physics system or desynced assets, so using a local member like this is probably fine..